src/test/suite/telemetry/connectionTelemetry.test.ts

```diff
@@ -1,17 +1,18 @@
 import { before, after, beforeEach, afterEach } from 'mocha';
 import { connect } from 'mongodb-data-service';
-import type { DataService } from 'mongodb-data-service';
 import { expect } from 'chai';
 import sinon from 'sinon';
+import type { DataService } from 'mongodb-data-service';
+
+import getCloudInfoModule from 'mongodb-cloud-info';
 
 import { ConnectionTypes } from '../../../connectionController';
-import { getConnectionTelemetryProperties } from '../../../telemetry/connectionTelemetry';
+import * as connectionTelemetry from '../../../telemetry/connectionTelemetry';
 
 const TEST_DATABASE_URI = 'mongodb://localhost:27018';
 
 suite('ConnectionTelemetry Controller Test Suite', function () {
-  this.timeout(8000);
-
+  this.timeout(20000);
```

Comment:
20000 ms sounds like a long time, what's causing this change? Maybe a comment would help future readers.

Reply:
Well, part of this test suite runs on the real connection. We have similar values in Compass for real-client testing. I will move it to the `'with live connection'` sub-suite to make it self-readable. Btw this is THE fluky test :)

Reply:
Sounds good 👍